doc/classes/PhysicsDirectSpaceState2D.xml

```diff
@@ -29,6 +29,24 @@
 				Returned points are a list of pairs of contact points. For each pair the first one is in the shape passed in [PhysicsShapeQueryParameters2D] object, second one is in the collided shape from the physics space.
 			</description>
 		</method>
+		<method name="get_complete_rest_info">
+			<return type="Dictionary[]" />
+			<param index="0" name="parameters" type="PhysicsShapeQueryParameters2D" />
+			<param index="1" name="max_results" type="int" default="32" />
+			<description>
+				Checks the intersections of a shape, given through a [PhysicsShapeQueryParameters2D] object, against the space. The information about the intersections are returned in an array containing dictionaries with the following fields:
+				[code]collider_id[/code]: The colliding object's ID.
+				[code]linear_velocity[/code]: The colliding object's velocity [Vector2]. If the object is an [Area2D], the result is [code](0, 0)[/code].
+				[code]normal[/code]: The collision normal of the query shape at the intersection point, pointing away from the intersecting object.
+				[code]depth[/code]: The collision depth of the intersection point.
+				[code]point[/code]: The intersection point.
+				[code]rid[/code]: The intersecting object's [RID].
+				[code]shape[/code]: The shape index of the colliding shape.
+				If the shape did not intersect anything, then an empty dictionary is returned instead.
+				The number of intersections can be limited with the [param max_results] parameter, to reduce the processing time.
+				[b]Note:[/b] This method does not take into account the [code]motion[/code] property of the object.
```

Comment:
This comment may be incorrect as per https://github.com/godotengine/godot/pull/105051/files#r2035580642 (only in 2D, not 3D).